### PR TITLE
Updated component to PRC

### PR DESCRIPTION
**Pull request by danielmoll**

### CODE REVIEW

File: src/get-dppx.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  const globalObject = global || (typeof window !== 'undefined' ? window : {});
+  const globalObject = global || (typeof window === 'undefined' ? {} : window);
   const dppxInDpi = 96;
   if ('devicePixelRatio' in globalObject) {
     return Number(globalObject.devicePixelRatio);
   } else if ('screen' in globalObject && 'deviceXDPI' in globalObject.screen) {
     return Math.round((
       Math.sqrt(globalObject.screen.deviceXDPI * globalObject.screen.deviceYDPI) / dppxInDpi
-    ) * 100) / 100;
+    ) * 100) / 100; // eslint-disable-line
```
Comment (keithamus) [inline src/get-dppx.js]: We shouldn't disable the eslint errors here. Instead give the two numbers variable names - to help readers of the code figure out what is going on here.

File: src/index.js [hunk 2]
```diff
@@ -1,30 +1,27 @@
...
+    const currentImageRatio = Math.abs(currentSource.width / currentSource.height);
+    const isPortrait = currentImageRatio < portraitImageRatioCutoff;
+    return isLessWide && isPortrait ? currentSource : previousSource;
+  });
+}
 
+export default class Picture extends React.Component {
   constructor({ sources }) {
-    super(...arguments);
+    super(...arguments); //eslint-disable-line
```
Comment (keithamus) [inline src/index.js]: Again, dont disable eslint. In this case just rewrite this as...

```js
constructor(props, ...rest) {
  super(props, ...rest);
  const { sources } = props;
}
```

**Comment by keithamus** [comment]
Just above notes, otherwise I'm happy with this.

File: src/get-dppx.js (hunk 1 above)
Comment (danielmoll) [inline src/get-dppx.js]: Forgot to remove that (as with the one below) but, while I agree in some cases having named variables in place of magic numbers isn't always as readable. This is a simple maths calculation and I feel that using the number directly is more obvious than a named variable. For example:

``` javascript
const divisor = 100;
return Math.round((Math.sqrt(globalObject.screen.deviceXDPI * globalObject.screen.deviceYDPI) / dppxInDpi) * divisor) / divisor;
```
vs
``` javascript
return Math.round((Math.sqrt(globalObject.screen.deviceXDPI * globalObject.screen.deviceYDPI) / dppxInDpi) * 100) / 100;
```

Or perhaps there's a better name for the variable?

File: src/index.js (hunk 2 above)
Comment (danielmoll) [inline src/index.js]: Aha, that's a useful example of `rest` parameters that eslint was complaining about :) Shall update.

**Pull request closed without merging**